MarkdownConverter/src/System/Tasks/ConversionRunner.ts

```diff
@@ -272,6 +273,34 @@ export class ConversionRunner
         let converter = new Converter(workspaceRoot, new Document(document, await this.LoadParser()));
         converter.Document.Quality = Settings.Default.ConversionQuality;
 
+        try
+        {
+            let headerPath = converter.Document.Attributes.HeaderPath;
+            if (!Path.isAbsolute(headerPath))
+            {
+                headerPath = Path.resolve(converter.WorkspaceRoot, headerPath);
+            }
+            converter.Document.Header.Content = Fs.readFileSync(headerPath, "utf8");
+        }
+        catch (err)
+        {
+            converter.Document.Header.Content = Settings.Default.HeaderTemplate;
+        }
+
+        try
+        {
+            let footerPath = converter.Document.Attributes.FooterPath;
+            if (!Path.isAbsolute(footerPath))
+            {
+                footerPath = Path.resolve(converter.WorkspaceRoot, footerPath);
+            }
+            converter.Document.Footer.Content = Fs.readFileSync(footerPath, "utf8");
+        }
+        catch
+        {
+            converter.Document.Footer.Content = Settings.Default.FooterTemplate;
+        }
```

Comment:
Please change this section to an if and check whether `converter.Document.Attributes.FooterPath` (for example using `"FooterPath" in converter.Document.Attributes`) and whether the file exists (for example by requiring `fs-extra` instead of `fs` and adding a code like this: `await FileSystem.pathExists(converter.Document.Attributes["FooterPath"]`).

`try`/`catch` might be a bad idea because this section could fail cause of other reasons than an inexistent file (for instance if permissions are denied).

My recommendation would be to make following if-statement:
```ts
if (
    ("FooterPath" in converter.Document.Attributes) &&
    await FileSystem.PathExists(converter.Document.Attributes["FooterPath"])
{
    // your code here
}